src/main/kotlin/io/bazel/worker/InvocationWorker.kt

```diff
@@ -23,7 +23,10 @@ class InvocationWorker(private val arguments: Iterable<String>) : Worker {
     runCatching {
       WorkerContext.run {
         doTask("invocation") { ctx -> execute(ctx, arguments) }.run {
-          println(log.out.toString())
+          val out = log.out.toString()
+          if (out != "") {
+            println(out)
```

Comment:
Would just calling `print` be a better solution here if we are just piping outputs so that the new lines aren't automatically appended?

Reply:
Works for me -- done

Reply:
Do you think it's worth checking whether the log ends with a new line, and if not, append one? In languages like Go, the logging libraries/functions do this automatically; not sure if there are similar conventions in kotlin.